Backend/router/v1/report.ctrl.js

```diff
@@ -0,0 +1,67 @@
+const models = require('../../models');
+
+const create = (req, res) => {
```

Comment:
Two space indent. others as well.
When you must use an anonymous function (as when passing an inline callback), use arrow function notation.
https://github.com/airbnb/javascript#arrow-functions

Reply:
modified it.

Reply:
@eunyyyoung  Please resolve a comment when it's done.